omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml

```diff
@@ -1,500 +1,735 @@
     </span>
     <!-- Beging of HTML Form -->
-
-    <div class="">
-
+    <div>
         <div class="row">
-            <div class="col-md-9 ">
-                <h3>Integrated Antanatal Register</h3>
+            <div class="col-md-6">
+                <h1>Integrated Antenatal Register</h1>
             </div>
-
-            <div class="col-md-3">
-                <submit class="confirm sm" submitLabel="Save" submitStyle="customSubmit" />
+            <div class="col-md-5"></div>
+            <div class="col-md-1">
+                <submit class="confirm" submitLabel="Save" submitStyle="customSubmit" />
             </div>
         </div>
         <div class="row">
             <div class="col-md-12">
-                <nav>
-                    <div class="nav nav-tabs" id="nav-tab" role="tablist">
-                        <a class="nav-item nav-link active" id="client-information-section-tab" data-toggle="tab" href="#client-information-section" role="tab" aria-controls="nav-client-nformation" aria-selected="true">
-                            Client Information
-                        </a>
-                        <a class="nav-item nav-link" id="investigations-tab" data-toggle="tab" href="#investigations-section" role="tab" aria-controls="nav-investigations" aria-selected="false">
-                            Investigations
-                        </a>
-                        <a class="nav-item nav-link" id="clinical-assessment-tab" data-toggle="tab" href="#clinical-assessment-section" role="tab" aria-controls="nav-clinical-assessment" aria-selected="false">
-                            Clinical Assessment
-                        </a>
+                <div class="card">
+                    <div class="card-header">
+                        <ul class="nav nav-tabs card-header-tabs">
+                            <li class="nav-item">
+                                <a class="nav-link active" data-toggle="tab" href="#client-information">Visit/Client Information</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" data-toggle="tab" href="#emtct-status">eMTCT Status</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" data-toggle="tab" href="#diagnosis-and-hiv-information">Diagnosis and HIV Information</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" data-toggle="tab" href="#investigations-section">Investigations</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" data-toggle="tab" href="#clinical-assessment-section">Clinical Assessment</a>
+                            </li>
```

Comment:
@semujju please rename this tab to treatment so that it corresponds to  the fields under it.

Reply:
Done